main.go

```diff
@@ -100,44 +195,131 @@ func main() {
 			LAPIStream:      lapiStream,
 			UpdateFrequency: conf.CloudflareConfig.UpdateFrequency,
 			Wg:              &wg,
+			UpdatedState:    stateStream,
+			CFStateByAction: states,
+			Count:           Count,
+		}
+		if *onlySetup {
+			workerTomb.Go(func() error {
```

Comment:
why not just running `SetUpCloudflareIfNewState()` (not in a goroutine) and then `return` ?